reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -442,11 +442,57 @@ <h2 id="rule-violation-summary">Rule Violation Summary (1 errors, 1 warnings, 0
           <tr class="ort-error" id="violation-1">
             <td><a href="#violation-1">1</a></td><td>rule 1</td><td>Maven:junit:junit:4.12</td><td>DETECTED: EPL-1.0</td><td>
               <p>EPL-1.0 error</p>
```

Comment:
I have several doubts about this approach:

1. The location where the how to fix instructions are put into the summary table.
Already the word summary gives a hint that its meant to provide an overview what is going on.
With those how-to fix it becomes quite bloated and getting an overview harder.
2. It is inside the error message column, thus if you want to look only at error messages it becomes harder to do that.
3. The scan result becomes much larger due to (a) redundancy of how to fix messages (b) verbose custom rules (c) not being limited to severity 
4. As a user I want a way to not see those how to fix messages at all

Proposed change which addresses all of the above points:
- have a separate table for fix me instructions e.g. at the bottom of the report (I don't neeed to read it)
- have deduplicated fixme instructions inside (report size, browser limits)
- only add a clickable link into the summary table (usability)

Reply:
1. For rule violations the summary is the only place where they are shown.
2. I have made the message collapsible, so that it only shows "How to fix" and requires a click to show the full message.
3. I think the effect on the size is negligible, e.g. for 1000 violations and 1k per message this would be ~1M, but a result file with 1000 violations should usually be far beyond that size already. But that can be tested.
4. Is collapsible good enough? Load the expected results file in your browser to test it.

The how to fix messages can contain content specific to a single rule violation, so I prefer to have them next to the violation instead of a separate table at the other end of the file. Browser limits have never been an issue with the static HTML report as far as I remember.